Stack outside of range error

Running into this error when I run Make:

`rgbasm -p 0xFF -isrc/ -isrc/include/ -Wall -Wextra -h -o obj/build_date.o src/res/build_date.asm
rgblink -p 0xFF -m bin/boilerplate.map -n bin/boilerplate.sym -o bin/boilerplate.gb obj/sgb.o obj/header.o obj/vectors.o obj/misc.o obj/crash_handler.o obj/unpb16.o obj/intro.o obj/build_date.o \
&& rgbfix -v -p 0xFF -v -i "BOIL" -k "HB" -l 0x33 -m 0x00 -n 0 -r 0x00 -t BOILERPLATE bin/boilerplate.gb
warning: Section "Stack"'s fixed address 0xdfc0 is outside of range [0xc000; 0xcfff]
warning: Section "Stack"'s end address 0xe000 is greater than last address 0xd000
error: Sanity checks failed
make: *** [Makefile:97: bin/boilerplate.gb] Error 1`

I haven't touched any of the files, this is right after I did a 'git clone "repo link" --recursive'. Both includes are there, and make clean doesn't help. RGBDS is v 0.4.0

--- a/README.md
+++ b/README.md
@@ -30,11 +30,13 @@ Edit `project.mk` to customize most things specific to the project (like the gam
 
 Everything in the `src` folder is the source, and can be freely modified however you want. The basic structure in place should hint you at how things are organized. If you want to create a new "module", you simply need to drop a `.asm` file in the `src` directory, name does not matter. All `.asm` files in that root directory will be individually compiled by RGBASM.
 
-There is "basic" code in place, but some things need your manual intervention. Look for `ld b, b` instructions, there will be comments indicating what to put.
+There is "basic" code in place, but some things need your manual intervention. Things requiring manual intervention will print an error message describing what needs to be changed, and a line number.
 
 The file at `src/res/build_date.asm` is compiled individually to include a build date in your ROM. Always comes in handy, and displayed in the bundled error handler.
 
 If you want to add resources, I recommend using the `src/res` folder. Add rules in the Makefile; there are several examples.
+
+It is recommended that the start of your code be in `src/intro.asm`.
 
 ## Compiling
 

--- a/src/header.asm
+++ b/src/header.asm
@@ -51,9 +51,9 @@ Reset::
 	dec b
 	jr nz, .copyOAMDMA
 
-	; You will want to init palettes here
+	FAIL "Edit to set palettes here"
 	; CGB palettes maybe, DMG ones always
-	ld b, b
+
 	; You will also need to reset your handlers' variables below
 	; I recommend reading through, understanding, and customizing this file
 	; in its entirety anyways. This whole file is the "global" game init,
@@ -104,7 +104,6 @@ Reset::
 	ldh [hOAMHigh], a
 
 	; `Intro`'s bank has already been loaded earlier
-	rst $38
 	jp Intro
 
 SECTION "OAM DMA routine", ROMX
@@ -144,9 +143,9 @@ wShadowOAM::
 	ds NB_SPRITES * 4
 
 
-; If using banked WRAM, then you will get an error; replace $E000 with $D000
+FAIL "If not using banked WRAM, then replace $D000 with $E000 and delete this line"
 ; This ensures that the stack is at the very end of WRAM
-SECTION "Stack", WRAM0[$E000 - STACK_SIZE]
+SECTION "Stack", WRAM0[$D000 - STACK_SIZE]
 
 	ds STACK_SIZE
 wStackBottom:

--- a/src/intro.asm
+++ b/src/intro.asm
@@ -2,4 +2,8 @@
 SECTION "Intro", ROMX
 
 Intro::
+; Remove this line
+	rst $38
+
+; Put your code here!
 	jr @

